lib/manager/terraform/util.ts

```diff
@@ -55,6 +55,9 @@ export function getTerraformDependencyType(
     case 'module': {
       return TerraformDependencyTypes.module;
     }
+    case 'terraform': {
+      return TerraformDependencyTypes.module;
+    }
```

Comment:
These three lines cause the existing extract test to fail. It seems to be matching some of the existing extracted dependencies.

Reply:
No longer applies. Have created a new manager